swagger-spec/registrations/node_links_list.yaml

```diff
@@ -0,0 +1,351 @@
+# /registrations/{node_id}/node_links
```

Comment:
Since node links are deprecated, let's not document them at all. Go ahead and delete this file.